azkaban-db/src/main/sql/create.version_set.sql

```diff
@@ -0,0 +1,9 @@
+CREATE TABLE version_set
+  (
+     num  INT NOT NULL auto_increment,
+     md5  CHAR(32) NOT NULL,
```

Comment:
Since the Md5 is always fixed to 32 bits, it is better to use char. Varchar is useful when fields are of varying size which varchar optimizes by using one extra bit.